.github/workflows/tests.yml

```diff
@@ -27,7 +34,16 @@ jobs:
         if: steps.npm-cache.outputs.cache-hit != 'true'
         run: npm ci
 
-      - name: Run tests
-        run: npx percy exec -- npm test
+      - name: Build assets
+        run: npm run build:assets
+
+      - name: Run test projects
+        run: |
+          npx jest --color --selectProjects "Snapshot tests"
+          npx jest --color --selectProjects "JavaScript unit tests"
+          npx jest --color --selectProjects "JavaScript behaviour tests"
+
+      - name: Run visual regression
```

Comment:
This is also running the component tests as well, right? Also not sure if '_Run_ visual regression' is the right name as the actual regression testing is a separate status check – this just sends the screenshots over to Percy.

Reply:
```suggestion
      - name: Send screenshots to Percy
```

Reply:
Yeah Percy starts a little service on `http://localhost:5338/percy/snapshot` then runs your command

Our tests call `await percySnapshot()` which sends the screenshot if Percy started cleanly (with a token)

Reply:
Thanks @36degrees I've updated this to your suggestion 👍